src/constants/skills.js

```diff
@@ -368,7 +379,15 @@ const icons = {
   elasticsearch:
     "https://www.vectorlogo.zone/logos/elastic/elastic-icon.svg",
   circleci:
-    "https://www.vectorlogo.zone/logos/circleci/circleci-icon.svg"
+    "https://www.vectorlogo.zone/logos/circleci/circleci-icon.svg",
+  scully:
+    "https://raw.githubusercontent.com/scullyio/scully/main/assets/logos/SVG/scullyio-icon.svg",
+  "11ty":
+    "https://www.11ty.dev/img/avatars/twitter/d4dc83e-90.webp",
+  sculpin:
+    "https://sculpin.io/css/sculpin-mascot-jackson-medium.png",
```

Comment:
please only add svg images

Reply:
Oh I couldn't find any svg images on their sites. 

Is there a platform you would want me to host it at? Or is a gist cool?